### PR TITLE
number of epochs was added and history in result dict was modified

### PR DESCRIPTION
**Pull request by VarshithaChennamsetti**
The number of epochs could be one of the items to analyze in the training result dictionary. Additionally, having a list of accuracy values for epochs could help us plot the training and validation accuracy and loss curves which could help in understanding if the model is overfitting or underfitting by examining the logs.

### CODE REVIEW

**Review by laszewsk — CHANGES_REQUESTED** [review]
likely a mistake as only indent is different, but original indent is correct.
should it be closed?

**Comment by VarshithaChennamsetti** [comment]
Yes, indent seems to be my mistake but I have removed [-1] from the 'history.history' values to indicate we want the accuracies for each epoch.

**Review by VarshithaChennamsetti — COMMENTED** [review]
Yes, indent seems to be my mistake but I have removed [-1] from the 'history.history' values to indicate we want the accuracies for each epoch. I also added num_of_epochs to the result dict.
